### PR TITLE
added link to SMTP Auth clients report

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/OfficeDocs-Support/issues/298

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @nam31, for your information

Thank you

File: microsoft-365/compliance/prepare-tls-1.2-in-office-365.md [hunk 1]
```diff
@@ -69,8 +69,9 @@ The following resources provide guidance to help make sure that your clients are
 - [New IIS functionality](https://cloudblogs.microsoft.com/microsoftsecure/2017/09/07/new-iis-functionality-to-help-identify-weak-tls-usage/) makes it easier to find clients on [Windows Server 2012 R2](https://support.microsoft.com/help/4025335/windows-8-1-windows-server-2012-r2-update-kb4025335) and [Windows Server 2016](https://support.microsoft.com/help/4025334/windows-10-update-kb4025334) that connect to the service by using weak security protocols.
 - Get more information about how to [solve the TLS 1.0 problem](https://www.microsoft.com/download/details.aspx?id=55266).
 - For general information about our approach to security, go to the [Office 365 Trust Center](https://www.microsoft.com/trustcenter/cloudservices/office365).
+- For identifying version of TLS used by SMTP clients, see [SMTP Auth clients insight and report in the Security & Compliance Center](/microsoft-365/security/office-365-security/mfi-smtp-auth-clients-report)
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/prepare-tls-1.2-in-office-365.md]:
```suggestion
- To identify the version of TLS used by SMTP clients, see [SMTP Auth clients insight and report in the Security & Compliance Center](/microsoft-365/security/office-365-security/mfi-smtp-auth-clients-report).
```

**Comment by yogkumgit** [comment]
@MaratMussabekov

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@MaratMussabekov - could you please commit my suggested changes? Many thanks.

**Comment by yogkumgit** [comment]
@MaratMussabekov Gentle reminder

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks

**Pull request merged**